client-python/grakn/service/Keyspace/KeyspaceService.py

```diff
@@ -0,0 +1,43 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import grpc
+
+from grakn.service.Keyspace.autogenerated.Keyspace_pb2_grpc import KeyspaceServiceStub
+import grakn.service.Keyspace.autogenerated.Keyspace_pb2 as keyspace_messages
+
+class KeyspaceService(object):
```

Comment:
Why is this class called `KeyspaceService`? The "service" is provided by the server (`grakn-engine`). The client "consumes" the service and provides the functionality to the user. This effectively serves as a `KeyspaceManager` and should probably be named that. But because we like to find a one-word name for these most outward facing APIs, @marco-scoppetta and I thought `Keyspaces` is a good compromise for a name. @flyingsilverfin @lolski

Reply:
Agreed to keep KeyspaceService